src/components/ConfirmDialog/ConfirmDialog.stories.ts

```diff
@@ -0,0 +1,37 @@
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import { ConfirmDialog } from './index'
+
+export default {
+  title: 'component/ConfirmDialog',
+  component: ConfirmDialog,
+  argTypes: {
+  }
+}
+
+const Template = (args, { argTypes }) => ({
+  vuetify: new Vuetify({ iconfont: 'mdi' }),
+  props: Object.keys(argTypes),
+  components: { ConfirmDialog },
+  template: '<confirm-dialog v-bind="$props" @hook:mounted="externalMount" />',
+  methods: {
+    async externalMount () {
+      const vm = this.$children[0] // target the component confirm-dialog
+      await Vue.nextTick() // wait that mounted() finished
+      vm.open(
```

Comment:
Oh wow!

Reply:
A cool little trick provided by the internet!